### PR TITLE
MWPW-129891: Checkout link bg color

### PR DESCRIPTION
**Pull request by yesil**
if a checkout link contains a strong text
or is a direct child of a STRONG element
make the button blue

Resolves: [MWPW-129891](https://jira.corp.adobe.com/browse/MWPW-129891)

Due to CORS, the **After URL** should be tested with the following CORS settings:

`Access-Control-Allow-Origin: *`
`Access-Control-Allow-Headers: x-api-key`

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/ilyas/merch?martech=off
- After: https://mwpw-129891--milo--yesil.hlx.page/drafts/ilyas/merch?martech=off

Screenshots:
image
image

### CODE REVIEW

**Comment by yesil** [comment]
@honstar good question. In the marquee they are bigger but by default they are smaller.
e.g: https://www.adobe.com/creativecloud.html

I'm going to update the PR and make them bigger only if inside a marquee.

**Comment by yesil** [comment]
@honstar  here how it looks.
image

**Review by VKniaz — APPROVED** [review]

File: libs/blocks/merch/merch.js [hunk 1]
```diff
@@ -94,7 +94,14 @@ window.tacocat.loadPromise = new Promise((resolve) => {
 function buildCheckoutButton(link, dataAttrs = {}) {
   const a = document.createElement('a', { is: 'checkout-link' });
   a.setAttribute('is', 'checkout-link');
-  a.setAttribute('class', 'con-button');
+  const classes = ['con-button'];
+  if (link.closest('.marquee')) {
+    classes.push('button-l');
```
[the remark this answers is not in the document]
Comment (yesil) [reply libs/blocks/merch/merch.js:99]: @rgclayton  @chrischrischris which ones is better:

A: merch block decides how large a button should be inside a marquee.
B: marquee decides how large a checkout link inside should be.

We should also think with other blocks in mind, ex: card, marquee, modal, caas, carousel.

IMO, keeping commerce related knowledge in merch.js and merch.test.js is easier than scattering it around in multiple blocks.

**Review by honstar — APPROVED** [review]

File: libs/blocks/merch/merch.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rgclayton) [reply libs/blocks/merch/merch.js:99]: @yesil This is really a Consonant question. The guidlines for buttons sizes in the marquee came from the Consonant team. 

However, depending on the marquee that's used those buttons sizes are already managed in the marquee. Using the default marquee block `marquee` with no variant you get `button-l`;

Here's all the current options:
https://main--milo--adobecom.hlx.page/docs/library/blocks/marquee

**Pull request merged**